stacks/kafka-flink/README.rst

```diff
@@ -1,89 +1,88 @@
-#####################################################
-Building IoT applications with open-source components
-#####################################################
+#######################################
+Apache Kafka, Apache Flink, and CrateDB
+#######################################
 
+*Building industrial IoT applications with open-source components.*
 
-***********************************************
-Part 1: Apache Kafka, Apache Flink, and CrateDB
-***********************************************
 
+*****
+About
+*****
 
 Introduction
 ============
 
-Kafka, Flink, and CrateDB are all distributed systems that provide elastic
-scaling, fault tolerance, and high-throughput, low-latency performance via
-parallel processing. Particularly, the use of CrateDB makes the stack an
-extremely good fit for handling industrial time-series workloads.
+`Apache Kafka`_, `Apache Flink`_, and `CrateDB`_, are each distributed systems
+that provide elastic scaling, fault tolerance, and high-throughput, low-latency
+performance via parallel processing. Particularly, the use of CrateDB makes the
+stack an extremely good fit for handling industrial time-series workloads.
 
 The front line of the stack is Apache Kafka, used to queue messages received
-from the IoT sensors and devices, making that data highly available to systems
-that need it.
+from IoT sensors and devices, making that data highly available to systems that
+need it.
```

Comment:
```suggestion
they need it.
```

Reply:
This sounds a bit "holprig" to me.

Reply:
Maybe then: `making that data highly available to other systems for further processing`?

Reply:
Excellent. Thanks!